contracts/interfaces/IRelayRecipient.sol

```diff
@@ -1,17 +1,17 @@
-pragma solidity ^0.5.16;
+pragma solidity ^0.6.2;
 
 /**
  * a contract must implement this interface in order to support relayed transaction.
  * It is better to inherit the BaseRelayRecipient as its implementation.
  */
-contract IRelayRecipient {
+abstract contract IRelayRecipient {
```

Comment:
why not interface?

Reply:
A) it was a contract
B) I think it has to call this._msgSender() for interface

Reply:
Interface can't declare"internal" methods

Reply:
Interface can't declare"internal" methods

Reply:
Interface can't declare"internal" methods